modules/lib/src/main/resources/assets/js/page-editor/LiveEditPage.ts

```diff
@@ -388,9 +391,11 @@ export class LiveEditPage {
 
         MoveComponentViewEvent.on(this.moveComponentViewRequestedListener);
 
+        const contentId = this.pageView?.getLiveEditParams().contentId;
+
         this.beforeContentSavedListener = (): void => {
-            this.updateSelectedPathInStorage(null);
-            this.updateSelectedTextCursorPosInStorage(null);
+            SessionStorageHelper.updateSelectedPathInStorage(contentId, null);
```

Comment:
```suggestion
            SessionStorageHelper.updateSelectedPathInStorage(contentId);
```

Reply:
No, updating without value gives impression that it is fetched elsewhere